scExpression/cell-type-ontology-ordering.md

```diff
@@ -0,0 +1,57 @@
+# Cell type ontology ordering
+
+Cell types in a dot plot can be ordered using lineage information to maximize the utility of the tool for pattern discovery and readability. The cell ontology (CL) directed acyclic graph (DAG) provides a good approximation to biological lineages. Thus we have a developed a method to create an ordered list of cell types based on the information contained in the CL DAG.
```

Comment:
I'm wondering if in the first line, you should say "utility of scExpression" instead of "utility of the tool" - in case someone followed a link from outside the documentation - maybe also link back to the main "scExpression page"?